Assets/Scripts/Minigame/DragToTarget.cs

```diff
@@ -40,22 +62,73 @@ public class DragToTarget : MinigameBaseClass
        
         // Get Left Mouse Click Input
         private bool _dragging => Input.GetMouseButton(0);

```

Comment:
This is on me if i missed this in a previous pull request, but this is still a property, plus a bool, so naming such as `IsDragging` would usually be preferred
Plus this is pretty sub-optimal. Ideally you want to check Input once a frame, and then cache it in a field, instead of running the same check in multiple places in code (see line 189/190)